src/ray/raylet/scheduling/cluster_task_manager.cc

```diff
@@ -417,6 +439,30 @@ void ClusterTaskManager::FillResourceUsage(
       by_shape_entry->set_backlog_size(backlog_it->second);
     }
   }
+
+  for (const auto &pair : infeasible_tasks_) {
+    const auto &scheduling_class = pair.first;
+    const auto &resources =
+        TaskSpecification::GetSchedulingClassDescriptor(scheduling_class)
+            .GetResourceMap();
+    const auto &queue = pair.second;
+    const auto &count = queue.size();
+
+    auto by_shape_entry = resource_load_by_shape->Add();
+
+    for (const auto &resource : resources) {
+      // Add to `resource_loads`.
+      const auto &label = resource.first;
+      const auto &quantity = resource.second;
+      (*resource_loads)[label] += quantity * count;
+
+      // Add to `resource_load_by_shape`.
+      (*by_shape_entry->mutable_shape())[label] = quantity;
+      // TODO (Alex): Technically being on `tasks_to_schedule` could also mean
+      // that the entire cluster is utilized.
+      by_shape_entry->set_num_infeasible_requests_queued(count);
+    }
```

Comment:
I think you need to add the backlog tracking code here (was probably merged after you created your branch).